funcbench/manifests/cluster.yaml

```diff
@@ -10,3 +10,5 @@ cluster:
         machinetype: n1-highmem-4
         imagetype: COS
         disksizegb: 100
+        labels:
+          node-type: funcbench-{{ .PR_NUMBER }}
```

Comment:
@rajdas98 The funcbench Makefile target that works with `cluster.yml` does not have the `PR_NUMBER` deployment var set, could you update the target `cluster_create` and `cluster_delete` for it? It'll be perfect then :)